oe_whitelabel.theme

```diff
@@ -31,10 +31,11 @@ function oe_whitelabel_preprocess_links__language_block(&$variables) {
     $label = $link['link']['#title'];
 
     $link = [
-      'href' => $href,
+      'path' => $href,
       'name' => $label,
       'id' => 'link_' . $language_code,
-      'hreflang' => $language_code,
+      'label' => $label,
+      'lang' => $language_code,
```

Comment:
if we are using 'lang' now then we should use the internal ids https://github.com/openeuropa/oe_theme/blob/3.x/oe_theme.theme#L656